FrontEnd/src/components/HeaderFooter/header/navbar/SearchBox.jsx

```diff
@@ -1,15 +1,29 @@
 import css from "./SearchBox.module.css"
```

Comment:
Btw, it is better to place CSS import as the last imported element if do not like to observe weird behaviour when smth overrides your imported style